patternfly/patternfly-elements
# [chore] Standardizing polyfills for reporting

## Standardizing polyfills for reporting

Preview: https://deploy-preview-647--happy-galileo-ea79c4.netlify.com/examples/

### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Moved polyfills out of the component js files into a separate `polyfills--<component-name>.js` to make reporting easier
* Standardized prefixing and added reference URLs
* Add TODO and Polyfill reporting to the Demo page

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Check there are no regressions in any components and that compiled assets still contain polyfills

#### Browser requirements
Your component should work in all of the following environments:


### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
